src/contracts/BMath.sol

```diff
@@ -62,16 +71,21 @@ contract BMath is BBronze, BConst, BNum {
   }
 
   /**
-   *
-   *    calcInGivenOut
-   *    aI = tokenAmountIn
-   *    bO = tokenBalanceOut               /  /     bO      \    (wO / wI)      \
-   *    bI = tokenBalanceIn          bI * |  | ------------  | ^            - 1  |
-   *    aO = tokenAmountOut    aI =        \  \ ( bO - aO ) /                   /
-   *    wI = tokenWeightIn           --------------------------------------------
-   *    wO = tokenWeightOut                          ( 1 - sF )
-   *    sF = swapFee
-   *
+   * @notice Calculate the amount of token in given the amount of token out for a swap
+   * @param tokenBalanceIn The balance of the input token in the pool
+   * @param tokenWeightIn The weight of the input token in the pool
+   * @param tokenBalanceOut The balance of the output token in the pool
+   * @param tokenWeightOut The weight of the output token in the pool
+   * @param tokenAmountOut The amount of the output token
+   * @param swapFee The swap fee of the pool
+   * @dev Formula:
+   * aI = tokenAmountIn
+   * bO = tokenBalanceOut               /  /     bO      \    (wO / wI)      \
+   * bI = tokenBalanceIn          bI * |  | ------------  | ^            - 1  |
+   * aO = tokenAmountOut    aI =        \  \ ( bO - aO ) /                   /
```

Comment:
I gotta get this off my chest 'cause otherwise it's gonna distract me all day:

It wouldn't be too hard to extend something like this: https://github.com/cag/sphinxcontrib-soliditydomain/ to add latex formulae in natspec and have them rendered in the docs without need to re-align an ascii diagram every time

Reply:
It's cool but I think is not a priority, we can add it later